panther_analysis_tool/backend/public_api_client.py

```diff
@@ -185,6 +193,32 @@ def bulk_upload(self, params: BulkUploadParams) -> BackendResponse[BulkUploadRes
 
         return to_bulk_upload_response(data)
 
+    def bulk_validate(self, params: BulkUploadParams) -> BulkUploadValidateStatusResponse:
```

Comment:
I realize that async bulk upload is the same, but will this just go forever if the status never changes (e.g. there's an error in a way that prevents the status from changing)?

Reply:
good call out I'll get a fix in for that

Reply:
fixed

Reply:
This is a good fix, but not what I was thinking of. I meant some sort of cap on retries, in case the status got stuck forever as NOT_PROCESSED

Reply:
ooh, that's enforced on the backend api. currently set at 20minutes. If we want to change that behavior I think that falls out of the scope of this PR